metricbeat/module/system/cpu/_meta/data.json

```diff
@@ -1,55 +1,41 @@
 {
-    "@timestamp": "2016-05-23T08:05:34.853Z",
-    "beat": {
-        "hostname": "host.example.com",
-        "name": "host.example.com"
-    },
-    "metricset": {
-        "host": "localhost",
-        "module": "system",
-        "name": "cpu",
-        "rtt": 115
-    },
-    "system": {
-        "cpu": {
-            "idle": {
-                "pct": 0.9492,
-                "ticks": 337165
-            },
-            "iowait": {
-                "pct": 0,
-                "ticks": 1026
-            },
-            "irq": {
-                "pct": 0,
-                "ticks": 0
-            },
-            "load": {
-                "1": 1.76,
-                "15": 0.56,
-                "5": 0.99
-            },
-            "nice": {
-                "pct": 0,
-                "ticks": 0
-            },
-            "softirq": {
-                "pct": 0,
-                "ticks": 78
-            },
-            "steal": {
-                "pct": 0,
-                "ticks": 0
-            },
-            "system": {
-                "pct": 0.0051,
-                "ticks": 7665
-            },
-            "user": {
-                "pct": 0.0457,
-                "ticks": 37296
-            }
-        }
-    },
-    "type": "metricsets"
-}
\ No newline at end of file
+  "@timestamp": "2016-07-27T11:06:46.743Z",
+  "beat": {
+    "hostname": "host.example.com",
+    "name": "host.example.com"
+  },
+  "metricset": {
+    "module": "system",
+    "name": "cpu",
+    "rtt": 29
+  },
+  "system": {
+    "cpu": {
+      "idle": {
+        "pct": 0.908
```

Comment:
Seems like we should enable the ticks for the generator just to have the full document with all options